### PR TITLE
layout adjustments

### PR DESCRIPTION
**Pull request by sammy-SC**
check please icon sizes because i had trouble exporting them from sketch.

### CODE REVIEW

File: Sample App/JTSImageVC/JTSImageVC/JTSViewController.m [hunk 1]
```diff
@@ -42,8 +42,7 @@ - (void)bigButtonTapped:(id)sender {
     imageInfo.image = self.bigImageButton.image;
 #endif
     imageInfo.title = @"A comfortable hotel with expansive views over the harbour and just next to a nice walkway which winds and undulates over the canal. What’s strange is";
-    imageInfo.dateText = @"8 April 2016";
-    imageInfo.timeText = @"8:36am";
+    imageInfo.dateText = @"8:36am, 8 April 2016";
```
Comment (famanson) [inline Sample App/JTSImageVC/JTSImageVC/JTSViewController.m:45]: Could we please remove these default texts?

**Comment by famanson** [comment]
@sammy-SC Icon sizes seem ok, what was the problem?

File: Sample App/JTSImageVC/JTSImageVC/JTSViewController.m (hunk 1 above)
Comment (sammy-SC) [inline Sample App/JTSImageVC/JTSImageVC/JTSViewController.m:45]: these are for testing. they are not default values.

**Comment by sammy-SC** [comment]
both of them were SVGs and I couldn't find a way to export them from Sketch.

File: Sample App/JTSImageVC/JTSImageVC/JTSViewController.m (hunk 1 above)
Comment (famanson) [inline Sample App/JTSImageVC/JTSImageVC/JTSViewController.m:45]: @sammy-SC oh I see. `Sample App`

**Comment by famanson** [comment]
@sammy-SC when you select the image, you can see on the bottom right corner there is an `Export` button. That's where you can get the PNG

**Comment by sammy-SC** [comment]
^^ me stupid. thanks. i was using the button in upper right corner.

**Pull request merged**